src/main/java/net/sf/jabref/importer/fileformat/MedlinePlainImporter.java

```diff
@@ -45,6 +46,8 @@
     private static final Pattern PMID_PATTERN = Pattern.compile("PMID.*-.*");
     private static final Pattern PMC_PATTERN = Pattern.compile("PMC.*-.*");
     private static final Pattern PMCR_PATTERN = Pattern.compile("PMCR.*-.*");
+    private static final Pattern CREATE_DATE_PATTERN = Pattern.compile("\\d\\d\\d\\d/[0123]?\\d/\\s?[012]\\d:[0-5]\\d");
+    private static final Pattern COMPLETE_DATE_PATTERN = Pattern.compile("\\d\\d\\d\\d\\d\\d\\d\\d");
```

Comment:
You can specify in regex how often a specific expression should match, instead of repeating it over and over.